packages/template-retail-react-app/app/pages/product-list/partials/color-refinements.jsx

```diff
@@ -46,6 +46,7 @@ const ColorRefinements = ({filter, toggleFilter, selectedFilters}) => {
                                     variant="outline"
                                     marginRight={0}
                                     marginBottom="-1px"
+                                    aria-label={`${value.label} (${value.hitCount})`} // @W-12627094@
```

Comment:
This should be an intl message!

Reply:
We get these values from the Shopper Search API which can provide the color name in a given locale, should this still be an intl message? 

![Screenshot 2023-11-13 at 11 19 14 AM](https://github.com/SalesforceCommerceCloud/pwa-kit/assets/88680517/26e9a257-1fbe-498e-aa26-bc60d91bc617)

Reply:
Yes - different locales might format the label/count differently, e.g. `Label (123)` vs `[123] Label`.